cmd/proxy/.env

```diff
@@ -1 +1,4 @@
 ATHENS_MONGO_STORAGE_URL=mongodb://127.0.0.1:27017
+GO_ENV=test_postgres
+POP_PATH=$PWD/cmd/proxy
+TMPDIR=$(mktemp -d)
```

Comment:
Sorry I missed this one. I this needs to be set in a shell script. It is only use in the unit test script.

Reply:
sure, here it is 92765c3